kythe/typescript/testdata/generic.ts

```diff
@@ -33,6 +33,14 @@ function generic<T>(x: T, y: IFace) {
   return x;
 }
 
+// Constrained generic.
+//- @IFace ref IFace
+function constrainedGeneric<T extends IFace>() {}
```

Comment:
(nit) maybe we should define the relationship between T and IFace as well?

Reply:
Good idea, fixed.